src/promptflow/promptflow/_sdk/entities/_flow.py

```diff
@@ -167,6 +168,11 @@ def _is_eager_flow(cls, data: dict):
         # If entry specified, it's an eager flow.
         return data.get("entry")
 
+    @classmethod
+    def _is_async_flow(cls, kwargs):
+        """Check if the flow is an async flow. Use field 'entry' to determine."""
```

Comment:
why say:  Use field 'entry' to determine

Reply:
Copied method, wrong in comment.